src/models/parallelCoordinates.js

```diff
@@ -38,11 +38,24 @@ nv.models.parallelCoordinates = function() {
             // Setup Scales
             x.rangePoints([0, availableWidth], 1).domain(dimensionNames);
 
+            var onlyNanValues = {};
             // Extract the list of dimensions and create a scale for each.
             dimensionNames.forEach(function(d) {
+                var extent = d3.extent(data, function(p) { return +p[d]; });
```

Comment:
Yes but here d is not really an index, it's more like a key because it's a string corresponding to the name, it's part of data. Maybe I should rename it k ?